client/lib/sharing/README.md

```diff
@@ -1,6 +1,8 @@
 Sharing
 =========
 
+**This is deprecated! Please use [`@automattic/request-external-access`](../../../packages/request-external-access) package instead.**
```

Comment:
This can be enforced via `eslint`. Check https://github.com/Automattic/wp-calypso/blob/master/.eslintrc.js#L135

Reply:
Enforced in https://github.com/Automattic/wp-calypso/pull/41990/commits/256fdfb7b8528829576034a5bf50079c5e9db1ac.